polyglot.js

```diff
@@ -220,7 +237,42 @@ class PolyGlot {
         }
     }
 
+    _getFontStyle(lang) {
+        return lang ? this.alphabets[this.tongues[lang]] : this.alphabets[this.tongues._default]
+    }
+
+    async loadLanguages(setting) {
+		const response = await fetch(`modules/polyglot/settings/${setting}.json`);
+		if (response.ok) {
+            const settingInfo = await response.json();
+            this.alphabets = settingInfo.alphabets;
+            this.tongues = settingInfo.tongues;
+            console.log(`Polyglot | Loaded ${setting}.json`);
+		} else {
+			console.error(`Failed to fetch ${setting}.json: ${response.status}`);
+			return;
```

Comment:
If there was an error in the download, I would suggest you fallback on setting `alphabets` and `tongues` with that 'default' case :
```
                this.alphabets = {common:'120% Dethek'}
                this.tongues = {_default:'common'}
```
I actually think you should set this before the fetch, because since fetch is async, there might be a moment where a chat message gets rendered before the fetch finished, so the two variables are not set and it could cause errors in the render function if the variable isn't set.
Setting the defaults before the fetch then overwriting them once we get the response would prevent that, but I'd worry about some people (on slow connections maybe) getting the messages rendered wrong on the initial load of foundry.

I wish the languages could be included from the start but I really like what you did with the json and dynamic loading, so I guess we just have that accept that possibility (which may never happen, I don't know 🤷), do set the defaults before the fetch though please.

Reply:
I believe the best fail-safe approach is to simply set the defaults in the constructor call, not only that its the most resilient. If we had a fallback on the loadLanguages function it could become an issue on future updates.

Reply:
Good idea, that works! :)